main.rb

```diff
@@ -1,10 +1,11 @@
-require_relative 'person'
-require_relative 'capitalize_decorator'
-require_relative 'trimmer_decorator'
+require_relative 'app'
 
-person = Person.new(22, 'maximilianus')
-puts person.correct_name
-capitalized_person = CapitalizeDecorator.new(person)
-puts capitalized_person.correct_name
-capitalized_trimmed_person = TrimmerDecorator.new(capitalized_person)
-puts capitalized_trimmed_person.correct_name
+def main
+  app = App.new
+  puts '-' * 50
+  puts "|\tWelcome to School Library App!\t\t |"
+  puts '-' * 50
+  app.run
+end
```

Comment:
I cannot run your app because you have an error related to your require statement in the `book.rb` file.

![image](https://user-images.githubusercontent.com/38088818/202860519-f6ee4689-6969-4a46-ac2a-ccbf2aa9dd58.png)

- Kindly make sure that you use small letters when you require `rental` instead of `Rental` to match the filename. Ruby is case sensetive.

Reply:
Hi @KDlamini , I have tested the program and it's working perfect. How come not working on your side?

Reply:
Hi @KDlamini , I checked the `book.rb` file and you were right, there's was a spelling mistake in the `require` statement. Thanks for your thorough feedback.